gdal*: add page

JakobMiksch opened the pull request:
This PR adds the page for `gdal2tiles.py` ([link](https://gdal.org/programs/gdal2tiles))

- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**
`3.2.2`

navarroaxel commented on pages/common/gdal2tiles.py.md:
```diff
@@ -0,0 +1,12 @@
+# gdal2tiles
```
Should match command name
```suggestion
# gdal2tiles.py
```

JakobMiksch replied:
Done.

JakobMiksch commented:
@navarroaxel thanks for your quick review. I changed the title in `gdal2tiles.py` as you suggested. I also fixed the errors discovered by the linter. Please let me know if there is more to do.

navarroaxel commented on pages/common/gdal2tiles.py.md:
```diff
@@ -0,0 +1,12 @@
+
+- Generate TMS tiles for the zoom levels 2-5 of a raster dataset:
+
+`gdal2tiles.py --zoom={{2-5}} {{input.tif}} {{output_folder}}`
```
We use `path/to/directory` as placeholder
```suggestion
`gdal2tiles.py --zoom={{2-5}} {{path/to/input.tif}} {{path/to/output_directory}}`
```

navarroaxel commented on pages/common/gdal2tiles.py.md:
```diff
@@ -0,0 +1,12 @@
+
+- Generate XYZ tiles for the zoom levels 2-5 of a raster dataset:
+
+`gdal2tiles.py --zoom={{2-5}} --xyz {{input.tif}} {{output_folder}}`
```
```suggestion
`gdal2tiles.py --zoom={{2-5}} --xyz {{path/to/input.tif}} {{path/to/output_directory}}`
```

navarroaxel commented on pages/common/gdal_contour.md:
```diff
@@ -0,0 +1,12 @@
+
+- Create a vector dataset with polygons every 100 meter:
+
+`gdal_contour -i {{100.0}} -p {{input.tif}} {{output.gpkg}}`
```
```suggestion
`gdal_contour -i {{100.0}} -p {{path/to/input.tif}} {{path/to/output.gpkg}}`
```

navarroaxel commented on pages/common/gdal_contour.md:
```diff
@@ -0,0 +1,12 @@
+
+- Create a vector dataset with contour lines every 100 meter and name the elevation property "ele":
+
+`gdal_contour -a {{ele}} -i {{100.0}} {{input.tif}} {{output.gpkg}}`
```
```suggestion
`gdal_contour -a {{ele}} -i {{100.0}} {{path/to/input.tif}} {{path/to/output.gpkg}}`
```

navarroaxel commented on pages/common/gdal_translate.md:
```diff
@@ -0,0 +1,20 @@
+
+- Convert raster dataset to different format:
+
+`gdal_translate -of {{"JPEG"}} {{world.tif}} {{output.jpeg}}`
```
The quotes should be outside the token syntax
```suggestion
`gdal_translate -of "{{JPEG}}" {{path/to/input.tif}} {{path/to/output.jpeg}}`
```

navarroaxel commented on pages/common/gdal_translate.md:
```diff
@@ -0,0 +1,20 @@
+
+`gdal_translate -a_srs {{EPSG:4326}} {{input.tif}} {{output.tif}}`
+
+- Reduce the size of a raster dataaset:
```
```suggestion
- Reduce the size of a raster dataset:
```

navarroaxel commented on pages/common/gdal_translate.md:
```diff
@@ -0,0 +1,20 @@
+
+- Convert a GeoTiff to a Cloud Optimzed GeoTiff:
+
+`gdal_translate {{input.tif}} {{output.tif}} -of COG -co COMPRESS=LZW`
```
```suggestion
`gdal_translate {{path/to/input.tif}} {{path/to/output.tif}} -of COG -co COMPRESS=LZW`
```

navarroaxel commented on pages/common/gdal_translate.md:
```diff
@@ -0,0 +1,20 @@
+
+- Reduce the size of a raster dataaset:
+
+`gdal_translate -outsize {{40%}} {{40%}} {{elevation.tif}} {{small.tif}}`
```
which is the input and output here?

navarroaxel commented on pages/common/gdaladdo.md:
```diff
@@ -0,0 +1,8 @@
+
+- Build overview images of a raster dataset with "average" resampling:
+
+`gdaladdo -r {{average}} {{input.tif}}`
```
```suggestion
`gdaladdo -r {{average}} {{path/to/file.tif}}`
```

navarroaxel commented on pages/common/gdaldem.md:
```diff
@@ -0,0 +1,16 @@
+# gdaldem
+
+> Tool to analyze and visualize digital elevation models(DEM).
```
```suggestion
> Tool to analyze and visualize DEM (digital elevation models).
```

navarroaxel commented on pages/common/gdaldem.md:
```diff
@@ -0,0 +1,16 @@
+
+- Create a hillshade from a DEM:
+
+`gdaldem hillshade {{input.tif}} {{output.tif}}`
```
```suggestion
`gdaldem hillshade {{path/to/input.tif}} {{path/to/output.tif}}`
```

navarroaxel commented on pages/common/gdalinfo.md:
```diff
@@ -0,0 +1,28 @@
+# gdalinfo
+
+> List various information about a GDAL supported raster dataset..
```
```suggestion
> List various information about a GDAL supported raster dataset.
```

navarroaxel commented on pages/common/gdalinfo.md:
```diff
@@ -0,0 +1,28 @@
+# gdalinfo
+
+> List various information about a GDAL supported raster dataset..
+> More information: <https://gdal.org/programs/gdalinfo.html#gdalinfo>.
```
```suggestion
> More information: <https://gdal.org/programs/gdalinfo.html>.
```

navarroaxel commented on pages/common/gdalinfo.md:
```diff
@@ -0,0 +1,28 @@
+
+`gdalinfo -hist {{input.tif}}`
+
+- List information about a Web Map Service(WMS):
```
Should be in mayus?
```suggestion
- List information about a web map service (WMS):
```

JakobMiksch replied:
Web Map Service (WMS) is an established term (see https://www.ogc.org/standards/wms) and should therefore be written in capital letters

navarroaxel commented on pages/common/gdalinfo.md:
```diff
@@ -0,0 +1,28 @@
+
+`gdalinfo WMS:{{https://services.meggsimum.de/geoserver/ows}}`
+
+- List information about a specific dataset of a Web Map Service(WMS):
```
```suggestion
- List information about a specific dataset of a web map service (WMS):
```

JakobMiksch replied:
Web Map Service (WMS) is an established term (see https://www.ogc.org/standards/wms) and should therefore be written in capital letters

navarroaxel commented on pages/common/gdalwarp.md:
```diff
@@ -0,0 +1,20 @@
+
+- Reproject a raster dataset:
+
+`gdalwarp -t_srs {{EPSG:4326}} {{input.tif}} {{output.tif}}`
```
```suggestion
`gdalwarp -t_srs {{EPSG:4326}} {{path/to/input.tif}} {{path/to/output.tif}}`
```

JakobMiksch commented:
@navarroaxel thanks for your review. I tried to apply all notes. Let me know if there is more to do.

CleanMachine1 reviewed (approved):
Don't know why this sat around for so long. Sorry for the wait

JakobMiksch commented:
meanwhile most of the code is merged

JakobMiksch commented on pages/common/gdal2tiles.py.md:
```diff
@@ -0,0 +1,12 @@
+# gdal2tiles.py
+
+> Generate TMS or XYZ tiles for a raster dataset.
+> More information: <https://gdal.org/programs/gdal2tiles.html>.
```
```suggestion
> More information: <https://gdal.org/programs/gdal2tiles>.
```

JakobMiksch commented:
all of the content has been merged by other PRs that's why it can be safely closed
